WordPress/src/main/java/org/wordpress/android/util/URLFilteredWebViewClient.java

```diff
@@ -0,0 +1,49 @@
+package org.wordpress.android.util;
+
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * WebViewClient that adds the ability of restrict URL loading (navigation) to a list of allowed URLs.
+ * Generally used to disable links and navigation in admin pages.
+ */
+public class URLFilteredWebViewClient extends WebViewClient {
+    ArrayList<String> allowedURLs = new ArrayList<>();
```

Comment:
Would make sense to use a `Set` instead of a `List` here since we only use it to check for `.contains()` later in `shouldOverrideUrlLoading`